ui/src/components/ResultBlock/index.tsx

```diff
@@ -1,11 +1,22 @@
 import React from "react";
+import styled from "styled-components";
 
-import "./styles.scss"
```

Comment:
Is there a reason to mix css and typescript code? Whats the benefit of using `styled-components`?

Reply:
Good question: 

It makes dynamic styling with React props really easy and makes composing/extending styles pretty clean. It also prevents annoying things in css like class name collisions, deep nesting, and overusing things like `!important`. More info here: https://styled-components.com/docs/basics

At the end of the day, it's a bit of a stylistic choice. All the FE projects at Stellar use it as a standard and I personally prefer writing TS for styled-components rather than dealing with scss and classnames